README.md

```diff
@@ -43,27 +43,29 @@ The governance of the project is described in more detail in the
 
 The Egeria team run three types of calls and all are welcome
 
-* **Developers call** - one hour - 
-  Discussion on code development - not minuted -
+* **Developers call** 
+   * one hour 
+   * Tuesday mornings (UK time) 
```

Comment:
Developers community call is on Thursdays right? and on Tuesdays it is focus group call.

Reply:
Yes our public meeting is Thursday am.

Reply:
My bad -corrected